privacy-protections/amp/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>AMP Links</title>
+
+  <style>
+      /* styles, if needed */
+  </style>
+</head>
+<body>
+  <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[AMP Links]</strong></p>
+  
+  <p>This test page has sample links to use for AMP link protection</P>
```

Comment:
Let's add an explanation how person testing can figure out if the right thing happened after clicking those links. Not sure if there is a general explanation, but if not, we can add something to each link like "should load vox.com instead of google.com".